docs/security.rst

```diff
@@ -37,13 +39,18 @@ either with `kafkastore.sasl.kerberos.service.name` or in the JAAS file.
 If the Schema Registry has a different service name than Kafka, at this time `zookeeper.set.acl` must be set to `false`
 in both the Schema Registry and Kafka.
 
-Upgrade from HTTP to HTTPS for REST API
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-By default Schema Registry ships with configs that allow you to make REST API calls over HTTP. The following configuration determine the protocol used by Schema Registry:
+.. _schema_registry_http_https:
+
+Configuring the REST API for HTTP or HTTPS
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default Schema Registry allows you to make REST API calls over HTTP. You may configure Schema Registry to allow either HTTP or HTTPS or both at the same time.
+
+The following configuration determine the protocol used by Schema Registry:
 
 ``listeners``
-  Comma-separated list of listeners that listen for API requests over either HTTP or HTTPS. If a listener uses HTTPS, the appropriate SSL configuration parameters need to be set as well.
+  Comma-separated list of listeners that listen for API requests over HTTP or HTTPS. If a listener uses HTTPS, the appropriate SSL configuration parameters need to be set as well.
 
   Schema Registry identities are stored in ZooKeeper and are made up of a hostname and port. If multiple listeners are configured, the first listener's port is used for its identity.
```

Comment:
This is no longer true and we must remove this.

Reply:
@mageshn , can you please clarify, specifically which part of this statement is no longer true?

Reply:
I think you mean that it's no longer either/or...that it can be both.  Is that right?

Reply:
@ybyzek that's true as well. But we can remove this statement ```Schema Registry identities are stored in ZooKeeper and are made up of a hostname and port. If multiple listeners are configured, the first listener's port is used for its identity.``` The reason we don't have to specify this is because it can now be either ZK or Kafka coordinator protocol and is probably an implementation detail.

Reply:
Thank you for clarifying.  I have now removed it.  Please let me know if good to merge.